code/game/objects/items/debugger.dm

```diff
@@ -0,0 +1,59 @@
+//The debugger! Can be used to test pretty much anything. Make a subtype with its own effects under debug().
+//Fun fact: It took 3 pull requests for this to finally get added in.
+
```

Comment:
I swear to fucking god if this PR doesn't get in I'm making one just for the debugger object.